Warmup-1/loneTeen/src/index.ts

```diff
@@ -0,0 +1,9 @@
+export function loneTeen(givenInput1: number, givenInput2: number): boolean {
```

Comment:
`givenInput`? Strange name, could you please name it to something more descriptive

Reply:
Renamed accordingly.